ui/src/components/RoleScopesTable/index.jsx

```diff
@@ -50,13 +50,16 @@ const sorted = pipe(
 }))
```

Comment:
Hm, I see a change to RoleScopesTable but not to ClientScopesTable. Do we need to feed the searchTerm prop to `ConnectionDataTable`?

Reply:
Added this now. Didn't add it at first because this view doesn't return anything regarding the search made.

<img width="1440" alt="Screenshot 2020-04-30 at 3 31 27 AM" src="https://user-images.githubusercontent.com/12775813/80651242-33aca700-8a93-11ea-8ea0-146fe27cb312.png">